src/misc/replayer.cc

```diff
@@ -0,0 +1,208 @@
+#include "bootstrap.h"
+#include "group.h"
+#include "utils.h"
+#include <string>
+#include <sys/syscall.h>
+ 
+using namespace std::chrono;
+int rcclReplayThreadIdx = -1;
```

Comment:
I just checked, and I think this should be
```suggestion
__thread int Replayer::rcclReplayThreadIdx = -1;
```